charts/handshake-node/values.yaml

```diff
@@ -0,0 +1,122 @@
+---
+# Network configuration (main for mainnet, testnet for testnet, regtest, etc.)
+handshake_network: main
+
+networkPorts:
+  main:
+    p2p: 12038
+    brontide: 44806
+    http: 12037
+    ns: 5349
+    rs: 5350
```

Comment:
Are these also client ports on the network? Typically, we don't do anything specific to an environment (or choose only one) and have the user override. Ports, etc. stay stable on the container itself, since each Pod gets a unique network address. However, if this is standard practice for the chain, then it's fine.